instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/NetClientAttributesExtractor.java

```diff
@@ -49,17 +50,33 @@ public void onEnd(
 
     internalSet(attributes, SemanticAttributes.NET_TRANSPORT, getter.transport(request, response));
 
-    String peerIp = getter.peerIp(request, response);
     String peerName = getter.peerName(request, response);
-
-    if (peerName != null && !peerName.equals(peerIp)) {
+    Integer peerPort = getter.peerPort(request, response);
+    if (peerName != null) {
       internalSet(attributes, SemanticAttributes.NET_PEER_NAME, peerName);
+      if (peerPort != null && peerPort > 0) {
+        internalSet(attributes, SemanticAttributes.NET_PEER_PORT, (long) peerPort);
+      }
     }
-    internalSet(attributes, SemanticAttributes.NET_PEER_IP, peerIp);
 
-    Integer peerPort = getter.peerPort(request, response);
-    if (peerPort != null && peerPort > 0) {
-      internalSet(attributes, SemanticAttributes.NET_PEER_PORT, (long) peerPort);
+    String sockPeerAddr = getter.sockPeerAddr(request, response);
+    if (sockPeerAddr != null && !sockPeerAddr.equals(peerName)) {
+      internalSet(attributes, AttributeKey.stringKey("net.sock.peer.addr"), sockPeerAddr);
+
+      Integer sockPeerPort = getter.sockPeerPort(request, response);
+      if (sockPeerPort != null && !sockPeerPort.equals(peerPort)) {
+        internalSet(attributes, AttributeKey.longKey("net.sock.peer.port"), (long) sockPeerPort);
+      }
+
+      internalSet(
+          attributes,
+          AttributeKey.stringKey("net.sock.family"),
+          getter.sockFamily(request, response));
+
```

Comment:
Shouldn't we have an if checking if the value isn't `inet`? I know we're kinda short-circuiting this in our getters (they simply never report `inet` socket family), but we probably can't expect everyone using instrumentation-api to do the same, especially when `inet` is a valid value according to the spec.

Reply:
I'm not sure, do we have any other examples of not passing along a semantic attribute value because it's the default value?

Reply:
Nvm, got confused. It's all okay 👍